redbadger/website-honestly
# Update events list banner

### Motivation

The webinar has happened so now we need to link to the recording. We open a new tab with the link instead of the modal because YouTube doesn't allow embedding standard YouTube links in an iframe and when embedding the playlist isn't visible which isn't deisred.

### Test plan

- Go to the events page and observe the new image for the banner
- Click the banner and observe that it opens a new tab with the YouTube recording

### Pre-merge checklist
